ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/BlurTransformation.kt

```diff
@@ -56,15 +57,15 @@ internal fun Bitmap.blur(context: Context, radius: Float, scaleDown: Boolean = t
     script.setInput(input)
     script.forEach(output)
 
-    val blurredBitmap = Bitmap.createBitmap(bitmap.width, bitmap.height, config)
+    val blurredBitmap = config?.let { createBitmap(bitmap.width, bitmap.height, it) }
```

Comment:
Looks like it doesn't accept a nullable value anymore... In this case, I'm making it so, if `config` is null, we return the original bitmap without the blurring. I don't fully understand in what situations this might be null though in my tests, it seems to work fine.